awx/ui_next/src/components/CodeMirrorInput/CodeMirrorInput.jsx

```diff
@@ -63,6 +64,7 @@ function CodeMirrorInput({
   readOnly,
   hasErrors,
   rows,
+  fullHeight,
```

Comment:
I assume this is not a required prop, right?  We should probably add this to prop types/default props below.

Reply:
It is good call.